crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -531,13 +533,15 @@ pub fn extract_clusters(
                 cluster_objects.spot_light_count as u32,
             ));
             for clusterable_entity in &cluster_objects.entities {
-                data.push(ExtractedClusterableObjectElement::ClusterableObjectEntity(
-                    *clusterable_entity,
-                ));
+                if let Ok(entity) = mapper.get(*clusterable_entity) {
```

Comment:
Shouldn't we modify the clusterizer system to store RenderEntity on Clusters directly, instead of using another query here?